ui/src/tasks/actions/index.ts

```diff
@@ -266,13 +287,17 @@ export const addTaskLabelsAsync = (taskID: string, labels: Label[]) => async (
   }
 }
 
-export const removeTaskLabelsAsync = (
-  taskID: string,
-  labels: Label[]
-) => async (dispatch): Promise<void> => {
+export const removeTaskLabelAsync = (taskID: string, label: Label) => async (
```

Comment:
renamed this to accurately reflect its functionality since only one label is ever passed